BraintreeCore/src/main/java/com/braintreepayments/api/AnalyticsEventBlob.kt

```diff
@@ -0,0 +1,17 @@
+package com.braintreepayments.api
+
+import androidx.room.ColumnInfo
+import androidx.room.Entity
+import androidx.room.PrimaryKey
+
+// Ref: https://www.reddit.com/r/javascript/comments/10st04/what_is_a_json_blob/
+@Entity(tableName = "analytics_event_blob")
+data class AnalyticsEventBlob(
+    @ColumnInfo(name = "json_string") val jsonString: String
+) {
+    // NOTE: default 0 value for _id is replaced when key is auto generated
+    @JvmField
+    @PrimaryKey(autoGenerate = true)
+    @ColumnInfo(name = "_id")
+    var id: Long = 0L
```

Comment:
Is there a reason why the `id` property cannot live in the main constructor and be a `val`?

https://developer.android.com/training/data-storage/room/defining-data

Reply:
That's a good call I'll update this. I can't remember an exact reason why this is `var`.

Reply:
I tried changing it to `val` and the build errors out. It may have to do with the version of Room we're using.

Reply:
Ok I found [this example](https://github.com/android/compose-samples/blob/c844e536deffe17799f73431271cfa2cddc7e0b0/Jetcaster/core/data/src/main/java/com/example/jetcaster/core/data/database/model/Category.kt#L24). I'll move the `id` declaration into the constructor 👍